docs/pages/guides/apple-privacy.mdx

```diff
@@ -0,0 +1,69 @@
+---
+title: Privacy Manifests
+description: Learn about configuring iOS privacy manifests for your mobile app.
+---
+
+import { ConfigReactNative } from '~/components/plugins/ConfigSection';
+
+If you're using a native iOS library that uses a "restricted reason" APIs, you'll need to configure an iOS privacy manifest to declare why you're including native code calling those APIs.
+
+More details and a list of "required reason" APIs can be found in the [Apple Developer Documentation](https://developer.apple.com/documentation/bundleresources/privacy_manifest_files).
+
+> **info** The information and steps included in this document are still in development and may change, either due to new tools built for this purpose or new requirements from Apple.
```

Comment:
```suggestion
> **info** The information and steps included in this guide are still in development and may change due to new tools built for this purpose or new requirements from Apple.
```
- Let's make the message of this callout a bit more clear.
- I think we could move the callout at the top of this page because it conveys this message as the first thing when a user visits this page and hence does not miss it.